package.json

```
@@ -45,11 +45,11 @@
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.12.5",
-    "@testing-library/dom": "^8.0.0"
+    "@testing-library/dom": "^8.0.0",
+    "@types/react-dom": "^17.0.0"
```

```suggestion
    "@types/react-dom": "*"
```
This matches how `@types/*` packages declare their type dependencies. We definitely want to add16.9 here since this library can be used with 16.x. And 18.x will also work. But I'd just go with `*` for consistency.